src/unitxt/metrics.py

```diff
@@ -291,7 +354,7 @@ def prepare(self):
         super().prepare()
         self.metric = evaluate.load(self.hf_metric_name, experiment_id=self.experiment_id)
 
-    def compute(self, references: List[List[str]], predictions: List[str]) -> dict:
+    def compute(self, references: List[List[Any]], predictions: List[Any]) -> dict:
```

Comment:
same q: why were the type hints changed?

Reply:
Same as before.  processed predictions  and references are not always string. (e.g. in NER)